recipes/qoi/all/test_package/test_package.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdint.h>
+
+#define QOI_IMPLEMENTATION
+#include "qoi.h"
+
+int main(void) {
```

Comment:
I would simplify the implementation by a lot - we don't intend to test functionality at all, just integration that the library can be included.

`#include "qoi.h"` -> if this works it tends to be enough

we may want to refer to a symbol or something defined, for example we can print or access the contents of `qoi_padding` - that's enough for the test.

Reply:
@jcar87 yeah, I might have done a little overboard on that one... 😅

Reply:
Thanks so much for addressing it! :)